ArduCopter/mode.h

```diff
@@ -1019,7 +1019,17 @@ class ModeRTL : public Mode {
     // for reporting to GCS
     bool get_wp(Location &loc) override;
 
-    RTLState state() { return _state; }
+    // RTL states
+    enum class State {
```

Comment:
Let's leave the class called RTLState.  "State" is just a bit too general..

Reply:
Being `enum class` means that outside `ModeRTL` users have to say (e.g.) `ModeRTL::State::FinalDescent`, so there's absolutely no confusion there.

Within the class the context of `State` should be obvious.  We could still prepend `RTL::` to it if you think it does require clarification, 'though?

Reply:
I'd prefer if the class was called RTLState rather than State.  So not RTL::State but just keep it's original name. sorry to be annoying.